README.md

```diff
@@ -130,15 +130,15 @@ This will prompt you to confirm removing old indices and setting up mappings on
 the Elasticsearch service (say `YES`). It will then proceed to rsync a partial
 CPAN in `/CPAN` for its metadata to be imported.
 
-Once the above is done, you should be able to see your local partial CPAN data
-in e.g. [http://localhost:5001/recent](http://localhost:5001/recent) and
-elsewhere.
-
-Alternatively, if you just want to hack on the web frontend, you can run this
-instead of all the above:
+After the initialization above completes, the next step is to start the web
+frontend with the following command:
 
     docker compose up web-server
 
+Once that is done, you should be able to see your local partial CPAN data
+in e.g. [http://localhost:5001/recent](http://localhost:5001/recent) and
+elsewhere.
+
```

Comment:
It has been a while since I set up new containers. After `docker compose up web-server` are you getting the data from your partial index or is it actually routing your requests to the live API? I suspect it's the latter and if that's the case, that's an error in our documentation.

Reply:
Actually, I think you're right: I'm seeing full recent history and I'm able to browse the "Crop" release which I don't think was imported. How do I make the web service connect to the local API?

Reply:
It looks like we are recommending this way: https://github.com/metacpan/metacpan-docker?tab=readme-ov-file#binmetacpan-localapi

It might be worth looking under the hood to see if this is still works and is the best idea. I would say if it does work, maybe we can just move on to the next thing rather than tweaking too much.